routes/index.js

```diff
@@ -416,7 +488,7 @@ router.get('/uuid-gen', (req, res) => {
 });
 
 /* GET new-job - visit new job page */
-router.get('/new-job', (req, res) => {
+router.get('/new_job', (req, res) => {
```

Comment:
Should ensure that only admins can access these (auth.ensureUserIsAdmin or something like that)